proposals/csharp-10.0/caller-argument-expression.md

```diff
@@ -148,6 +148,7 @@ There should always be an expression corresponding to the `this` parameter. Even
 - Multiple parameters marked with `CallerArgumentExpression` are permitted, as shown above.
 - The attribute's namespace will be `System.Runtime.CompilerServices`.
 - If `null` or a string that is not a parameter name (e.g. `"notAParameterName"`) is provided, the compiler will pass in an empty string.
+- The type of the parameter `CallerArgumentExpressionAttribute` is applied to must have a standard implicit conversion from `string`. This means no user-defined conversions from `string` are allowed, and in practice means the type of such a parameter must be `string`, `object`, or an interface implemented by `string`.
```

Comment:
I think "standard conversion" is the correct term (as it originally was). My comment was saying that specs for other caller info mention "standard implicit conversion" incorrectly. I also added "dynamic"

```suggestion
- The type of the parameter `CallerArgumentExpressionAttribute` is applied to must have a standard conversion from `string`. This means no user-defined conversions from `string` are allowed, and in practice means the type of such a parameter must be `string`, `object`, `dynamic`, or an interface implemented by `string`.
```

Reply:
Dynamic is just object with a different name, I don't think it needs to be called out.